lib/utils/format.ts

```diff
@@ -105,3 +106,24 @@ ${country || ''}
 ${postalCode}
   `;
 };
+
+export const formatPaymentType = (paymentMethod: string) => {
```

Comment:
Please add a JSDoc comment to this function, similar to how it's done for the other formatting functions in this file.

Also, please add a return type:
```suggestion
export const formatPaymentType = (paymentMethod: string): string => {
```

Reply:
Still missing JSDoc. Please see other formatting functions in this file